elastic/elastic-agent
# Remove cloud-defend from agent

## What does this PR do?



cloud-defend (a.k.a. Defend for Containers) will be removed from the next major release. This removes cloud-defend from agent, by removing it from packaging scripts and template kubernetes files.

## Why is it important?



## Checklist



- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation

## Disruptive User Impact



Defend for Containers never left beta, so although removing it could be disruptive to users of the integration, it's acceptable to be a breaking change in a major update.

## How to test this PR locally

After building the agent package, inspect it to ensure the cloud-defend package is not included in it, and the cloud-defend executable isn't on the system after installing the agent.
